File: .matrix223-python-let-candidate.xU9hBY/engines/polyglot-route-engine/fixtures/module/python/equivalence_module.py
def calculate(subtotal: int, tax: int) -> int:
    if subtotal < 0:
        return 0
    return subtotal + tax


def clamp(value: int, minimum: int, maximum: int) -> int:
    if value < minimum:
        return minimum
    if value > maximum:
        return maximum
    return value


def difference(left: int, right: int) -> int:
    return left - right


def clampNumber(value: float, minimum: float, maximum: float) -> float:
    if value < minimum:
        return minimum
    if value > maximum:
        return maximum
    return value


def both(left: bool, right: bool) -> bool:
    return left and right

File: .matrix223-python-let-candidate.xU9hBY/engines/polyglot-route-engine/fixtures/module/rust/equivalence_module.rs
fn calculate(subtotal: i64, tax: i64) -> i64 {
    if subtotal < 0 { return 0; }
    subtotal + tax
}

fn clamp(value: i64, minimum: i64, maximum: i64) -> i64 {
    if value < minimum { return minimum; }
    if value > maximum { return maximum; }
    value
}

fn difference(left: i64, right: i64) -> i64 { left - right }

fn clampNumber(value: f64, minimum: f64, maximum: f64) -> f64 {
    if value < minimum { return minimum; }
    if value > maximum { return maximum; }
    value
}

fn both(left: bool, right: bool) -> bool { left && right }
